ruby/vampires.rb

```diff
@@ -0,0 +1,152 @@
+
+def vampires
+
+puts "Welcome to Immortal Life Technologies"
+puts""
+
+counter = 0
+puts "How many of the following questionnaires will be administered?"
+total_employees = gets.chomp.to_i
+	until counter == total_employees
+	puts "Please fill out the following questionnaire to better serve your needs."
+	puts ""
+		
+	puts "What was you full name given at the time of your birth?"
+	birth_name = gets.chomp.capitalize
+	puts ""
+		
+	puts "How many years have you roamed this planet?"
+	age = gets.chomp.to_i
+	puts ""
+
+	puts "What was the year of your birth?"
+	birth_year = gets.chomp.to_i
+	puts ""
+
+# Mortal Age Confirmation #
+
+	mortal_age = false
+	until mortal_age
+
+#	puts "How many years have you roamed this planet?"
+#	age = gets.chomp.to_i
+
+#	puts "What was the year of your birth?"
+#	birth_year = gets.chomp.to_i
+
+		current_year = 2017
+		if age == (current_year - birth_year)
+			mortal_age = true
+		else
+			mortal_age = false
+		end
+	end
+
+# Garlic Aversion Confirmation #
+
+	garlic_choice = nil
+	while garlic_choice == nil
+
+	puts "Today the cafeteria is serving spaghetti. Would you care for some garlic bread? (y/n)"
+	garlic = gets.chomp.downcase
+	puts ""
+
+		if garlic == "yes" || garlic == "y"
+			garlic_choice = true
+		elsif garlic == "no" || garlic == "n"
+			garlic_choice = false
+		else
+			garlic_choice = nil
+		end
+	end
+
+# Immortality Confirmation #
+
+	mortality = nil
+	while mortality == nil
+
+	puts "Would you care to enroll in the Company's Health Insurance plan?"
+	health = gets.chomp
+	puts ""	
+
+		if health == "yes" || health == "y"
+			mortality = true
+		elsif health == "no" || health == "n"
+			mortality = false
+		else
+			mortality = nil
+		end
+	end
+
+# Vampire Booleans #
+
+#	V1 = vampire 
+
+    vampire = nil
+	while vampire == nil
+
+		if name == "Drake Cula"
+			puts "#{birth_name.capitalize} is definitely a vampire!"
+			vampire = true
```

Comment:
Instead of having a `vampire` variable that returns true or false, you can set that variable to string result we're expecting, like `vampire = "Definitely a vampire"` and that variable would be updated as the program went on in case later conditions evaluated to true.